config/jsonconfig/config.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I was trying to follow the guidelines for godocs:
https://blog.golang.org/godoc-documenting-go-code

But, rereading it, I realize I missed the convention to put the comment into doc.go; so I've moved it there.

I'm following specifically the example they give, which points me to https://golang.org/src/encoding/gob/doc.go , which uses // for copyright lines but then /* */ for the long comment in doc.go, even though the rest of the comments in the package use //

Does that make sense?